zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/DocumentenApiPlugin.kt

```diff
@@ -120,7 +120,7 @@ class DocumentenApiPlugin(
             inhoudAsInputStream = contentAsInputStream,
             beschrijving = description,
             informatieobjecttype = informatieobjecttype,
-            storedDocumentUrl = storedDocumentUrl
+            storedDocumentKey = storedDocumentUrl
```

Comment:
url to key? why is storedDocumentUrl named that way still? should it not be storedDocumentKey

Reply:
That is a good question. I'll change that name as well